package.json

```diff
@@ -57,7 +58,7 @@
   },
   "packageManager": "pnpm@9.6.0",
   "engines": {
-    "node": "^18.20.1",
+    "node": ">=18.0.0 <21.0.0",
```

Comment:
fwiw I see this most often formatted like `18.x || 20.x`

Reply:
that would not include v19, is that the intention?

Reply:
i think the semantic difference here is that `18.x || 20.x` ignores `19.x`, do we need to ignore it?

Reply:
we can test if it works with `19.x` by pushing a commit with `19.x` and checking that CI is still happy, if it works i'm in favor of including it

Reply:
mmm good point, not sure why I've seen it this way - maybe because the even release are considered LTS

Reply:
yeah, looking more into it, looks like that's the reason. I'd be more inclined to use what seems more conventional which is `18.x || 20.x`. It still doesn't stop `19.x` from being used, just with a warning.

Reply:
> even-numbered releases are maintained for longer and are considered more stable, making them better for production environments

wow TIL